docs/01-overview/README.md

```diff
@@ -8,9 +8,22 @@ Kyma is an application runtime that provides you a flexible and easy way to conn
 
 ![overview](./assets/kyma-overview.svg)
 
+## Modular Kyma
+
+Classic Kyma offered a fixed set of preconfigured components whose development rhythm is synchronized and determined by the release schedule. With the modular approach, Kyma components become modules, each providing one functionality developed independently of the other ones. With each successive release, fewer components will be available within the preconfigured Kyma, but more and more will be offered as independent modules.
```

Comment:
I ask to myself is how to install Kyma. I find article under get started. But apparetnly it didn’t install both modules. I need a quidance how to do it. Probably it should be mentioned on quick install page (not only on a page  dedicated for  module).

Reply:
I created this PR to complete the task - https://github.com/kyma-project/kyma/pull/17715. PTAL.